pages/visit/why-abu-dhabi.jsx

```diff
@@ -3,8 +3,10 @@ import { Container } from "@mui/material";
 import EvLayout from "components/layouts/EvLayout";
 import api from "utils/api/grocery3-shop";
 import Section1 from "../../src/pages-sections/why-abu-dhabi/Section1";
```

Comment:
you can use global imports ... no need  to do ../../srce/pages-sections
you can use "pages-section/" directly no 
this way we can easily copy the imports later